content/en/docs/concepts/configuration/configmap.md

```diff
@@ -236,8 +290,8 @@ spec:
         configMapKeyRef:
           name: myconfigmap
           key: username
-
 ```
+If you add this Pod to your cluster, you will see that only CONFIGMAP_USERNAME is set as an environment variable and it has the value of the username key from the myconfigmap ConfigMap.
```

Comment:
It would be great to delimit code-blocks from the text by new line ;-)

```suggestion

If you add this Pod to your cluster, you will see that only CONFIGMAP_USERNAME is set as an environment variable and it has the value of the username key from the myconfigmap ConfigMap.
```